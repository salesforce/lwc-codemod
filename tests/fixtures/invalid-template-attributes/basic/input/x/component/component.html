<template lwc:render-mode="light" class="foobar">
    <div>
        <template onclick={handleClick}>click me!</template>
    </div>
</template>