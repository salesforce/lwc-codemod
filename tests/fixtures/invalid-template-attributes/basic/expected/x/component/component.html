<template lwc:render-mode="light">
    <div>
        <template>click me!</template>
    </div>
</template>